rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction.adoc

```diff
@@ -0,0 +1,107 @@
+# Usecase: Extract Data from a Legacy System
+
+Extracting data from legacy systems is a common use case for RPA. 
```

Comment:
Suggest giving some examples of when you want to extract data from a legacy system - and what you might do with it.

Reply:
done